Update extractcode --shallow option description (#2959)
nexB/scancode-toolkit: docs/source/tutorials/how_to_run_a_scan.rst

.. note::

>     ``--shallow`` option can be used to prevent recursive extraction of nested archives. 

Reviewer: ```suggestion
    Use the ``--shallow`` option to prevent recursive extraction of nested archives.
```

Author: Done!
